benefits/static/css/styles.css

```diff
@@ -420,10 +419,6 @@ footer.global-footer .footer-links a:active {
   padding-top: 20vh;
 }
 
-.logged-out.enrollment-success .container.content h1 {
```

Comment:
Just FYI I deleted all these `logged-out` styles in my Enrollment fixes in #1012